blocks/components/editable/index.js

```diff
@@ -291,9 +360,7 @@ export default class Editable extends wp.element.Component {
 	}
 
 	changeFormats( formats ) {
-		if ( this.state.bookmark ) {
-			this.editor.selection.moveToBookmark( this.state.bookmark );
-		}
+		this.editor.focus();
```

Comment:
I'm seeing a bug when creating links where the first time I add a link and edit it right after. The link is not saved and the caret is moved to the beginning of the block. I suspect the change here to be responsible of this.